packages/@aws-cdk/aws-cloudformation/test/integ.core-custom-resources.ts

```diff
@@ -0,0 +1,31 @@
+import { App, CfnOutput, Construct, CustomResource, CustomResourceProvider, CustomResourceProviderRuntime, Stack, Token } from '@aws-cdk/core';
+
```

Comment:
I like to put a stack verification steps comment block here so that one doesn't have to read the integration test to understand when it actually worked - https://github.com/aws/aws-cdk/blob/master/CONTRIBUTING.md#integration-tests

In this case, I suppose the verification steps are that the output of 'GetAtt.Attribute1' is 'foo' and 'GetAtt.Attribute2' is 1234.

Reply:
Yes, I also had a similar idea. Basically allow to specify assertions against the CFN outputs would be awesome.

Reply:
This has not been addressed.

Reply:
I thought you wanted to add mechanical support for this in `cdk-integ` but okay, I can add some commentary.

Reply:
added